### DISCUSSION TITLE
My application for Access free GitHub Education benefits is still rejected

### DISCUSSION THREAD

**Discussion by zakscodeworld** [community/community#140141] [New to GitHub]
Hi everyone,

I hope this message finds you well. I have applied for the GitHub Education Status three times now, but unfortunately, my application has not been approved. Despite carefully submitting all the required documents and ensuring that my profile settings are complete, my request has been denied.

Here are the steps I have taken:

Uploaded all necessary documents (e.g., enrollment certificate, student ID)
Completed and updated my GitHub profile
Verified all information for accuracy
Since I have met all the necessary requirements, I am unsure why my application has been rejected. I would greatly appreciate it if anyone could provide insights or advice on what might be going wrong or if there are any additional steps I should take.

Thank you very much for your support and time.
![Screenshot 2024-09-29 223136](https://github.com/user-attachments/assets/eb62ece0-c4ec-4917-9051-c6e85307bde3)

**Comment by Bashamega** [comment]
I think, you should set your name on your github profile to match the name in your documents.

**Reply by zakscodeworld** [reply]
Thank you very much
